app/src/main/java/com/codesoom/assignment/application/AuthenticationService.java

```diff
@@ -0,0 +1,54 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.domain.UserRepository;
+import com.codesoom.assignment.dto.UserLoginData;
+import com.codesoom.assignment.errors.LoginNotMatchPasswordException;
+import com.codesoom.assignment.errors.UserNotFoundException;
+import com.codesoom.assignment.errors.InvalidAccessTokenException;
+import com.codesoom.assignment.utils.JwtUtil;
+import org.springframework.stereotype.Service;
+
+/**
+ * 인증, 인가와 관련된 처리를 담당합니다.
+ */
+@Service
+public class AuthenticationService {
+
+    private final JwtUtil jwtUtil;
+    private final UserRepository userRepository;
+
+    public AuthenticationService(JwtUtil jwtUtil, UserRepository userRepository) {
+        this.jwtUtil = jwtUtil;
+        this.userRepository = userRepository;
+    }
+
+    /**
+     * User 로그인 정보를 확인해 access token 발급합니다.
+     *
+     * @param userLoginData email과 password를 담은 User 정보
+     * @return access token
+     * @throws UserNotFoundException User 정보가 존재하지 않는 경우 던집니다.
+     * @throws LoginNotMatchPasswordException User의 password가 일치하지 않는 경우 던집니다.
```

Comment:
`User` 정보가 없는 것과 찾을 수 없는 것은 다릅니다. 컴퓨터에서 어떤 정보를 우리가 찾으려 할 때 그 정보가 조회되지 않았다고 해서 함부로 그 정보가 없다고 속단할 수 없습니다. 정전이 되었을 수도 있고, 순간적으로 인터넷이 끊어졌을 수도 있고, 그 정보가 다른 데이터베이스로 이전되었을 수도 있습니다. 우리가 할 수 있는 것은 그것이 없다고 단정짓는 것이 아니라 그것을 찾지 못했다고 인정하는 것 뿐입니다. 그래서 수많은 프로그램들의 에러 메시지에 "파일을 찾지 못했습니다." "주소를 찾지 못했습니다." 와 같은 내용이 들어있는 거고요.

```suggestion
     * @throws UserNotFoundException User 정보를 찾을 수 없는 경우
     * @throws LoginNotMatchPasswordException 로그인 시도에 사용한 password가 일치하지 않는 경우
```

Reply:
말씀하신 대로 찾습니다. 가 더 알맞은 내용 같습니다.